save-agent/src/linuxX64Main/kotlin/com/saveourtool/save/agent/SaveAgent.kt

```diff
@@ -82,6 +83,10 @@ class SaveAgent(private val config: AgentConfiguration,
             state.value = AgentState.BUSY
             sendDataToBackend { saveAdditionalData() }
 
+            logDebugCustom("Wil now download save-cli with version $SAVE_CORE_VERSION")
```

Comment:
agent downloads `SAVE_CLI_EXECUTABLE_NAME` here, maybe agent should set `AgentEnvName.CLI_COMMAND` instead of orchestrator?